src/PyCall.jl

```diff
@@ -98,9 +98,11 @@ it is equivalent to a `PyNULL()` object.
 """
 ispynull(o::PyObject) = o.o == PyPtr_NULL
 
-function pydecref_(o::Union{PyPtr,PyObject})
+function pydecref_(o::Union{PyPtr,PyObject}) :: Nothing
```

Comment:
We don't use the return value anywhere. So why not return `nothing` always?

Reply:
It seems better to return something potentially useful (`o`), so that the result can be chained with some other function.  Also, this way it mirrors `pyincref`.   Why not just return `o` always?